src/api/controllers/articleController.js

```diff
@@ -1,15 +1,14 @@
 import cloudinary from 'cloudinary';
 import models from '../models';
 import slugHelper from '../../helpers/slug.maker';
-import statusCodes from '../../helpers/constants/status.codes';
 import errorMessage from '../../helpers/constants/error.messages';
 import articleValidator from '../../helpers/validators/articleValidator';
+import status from '../../helpers/constants/status.codes';
 
 const { Article, Category } = models;
-const { CREATED } = statusCodes;
-
+const { CREATED } = status;
 /**
- * this is the article controller
+ *
```

Comment:
why did you remove this js doc comment, this will help anyone to understand what's going on down to the page he is on